### DISCUSSION TITLE
Creating Docker Container for Influx V3

### DISCUSSION THREAD

**Discussion by serrafabio** [influxdata/influxdb#24728] [Q&A]
Dear community,

I am currently developing a research project that involves creating local databases using software like Timescale and Influx V1 and V2. To achieve this, I use docker container solutions to generate a host and client that simulate machine communication, and a server that receives the data.

I have tested my project with Influx V3 using the cloud solution, and it worked fine. However, I am now searching for an open-source solution. I tried the solution presented in a certain repository, but when I launched it with the dockerfile provided, the container loaded and stayed with the message "binding listener." I am now unsure of what my next steps should be to create a docker container that runs Influx V3 as localhost.

Can anyone here help me with this issue? Your attention and help would be much appreciated. Thank you!

**Comment by pauldix** [comment]
We're not yet providing builds of open source 3.0, it's definitely not ready for testing yet. We'll provide alpha builds later this year with getting started documentation.

**Comment by finalyards** [comment]
@pauldix Mid-2026, looking for samples of InfluxDB v3 (open source) Docker use.
